modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -262,8 +274,11 @@ private void handleFinishTxCommand(FinishTxCommand cmd, long commandIndex) throw
 
         TxMeta txMetaToSet = new TxMeta(
                 stateToSet,
-                cmd.replicationGroupIds(),
-                cmd.commitTimestamp()
+                cmd.tablePartitionIds()
+                        .stream()
+                        .map(tpIdMsg -> (ReplicationGroupId) tpIdMsg.asTablePartitionId())
```

Comment:
Can this be done with method reference?

Reply:
Yes, method reference worked out, thanks!